CHANGELOG.md

```diff
@@ -9,6 +9,23 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Fix `SigSet` incorrect implementation of `Eq`, `PartialEq` and `Hash`
   ([#1946](https://github.com/nix-rust/nix/pull/1946))
 
+### Added
+
+- Added `impl From<Signal> for SigSet`.
```

Comment:
I don't know what happened here, but this sure is a riotous merge error.  Could you fix it please?

Reply:
Fixed.